src/Augmentation/AugmentationHelpers.jsx

```diff
@@ -1675,6 +1675,32 @@ function initAugmentations() {
 	// Aevum
     // TODO Later Something that lets you learn advanced math...this increases int
     // and profits as a trader/from trading
+    const PCMatrix = new Augmentation({
+        name:AugmentationNames.PCMatrix, repCost:1e4, moneyCost:1e7,
```

Comment:
You need to add the `PCMatrix` property to the `AugmentationNames` (`AugmentationNames.ts`) object, which as far as I can tell you haven't done here.